sim/paladin/avengers_shield.go

```diff
@@ -0,0 +1,59 @@
+package paladin
+
+import (
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+	"time"
+)
+
+func (paladin *Paladin) registerAvengersShield() {
+	if !paladin.hasRune(proto.PaladinRune_RuneLegsAvengersShield) {
+		return
+	}
+
+	// Avenger's Shield hits up to 3 targets.
+	results := make([]*core.SpellResult, min(3, paladin.Env.GetNumTargets()))
+
+	paladin.GetOrRegisterSpell(core.SpellConfig{
+		ActionID:    core.ActionID{SpellID: 407669},
```

Comment:
If things are set up right you should be able to use `int32(proto.PaladinRune_RuneLegsAvengersShield)` here instead of the raw ID int

Reply:
Done.